compiler/one-cmds/one-import-onnx

```diff
@@ -41,6 +41,69 @@ import onelib.utils as oneutils
 sys.tracebacklimit = 0
 
 
+# Class to rename input/output to prevent issues while import ONNX models
+class TidyIONames:
+    input_nodes = []
+    output_nodes = []
+    remap_inputs = []
+    remap_outputs = []
+    initializers = []
+    onnx_model = None
```

Comment:
Ah, another comment:
Is there any reason to use `class variables` instead of `instance variable` here?
These members will be shared by all instances of the class.

From https://docs.python.org/3.8/tutorial/classes.html#class-and-instance-variables

> Generally speaking, instance variables are for data unique to each instance and class variables are for attributes and methods shared by all instances of the class:

Reply:
Ah, should use class variables :)
Thanks for pointing this !